website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

```diff
@@ -47,3 +47,17 @@ dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructu
 * Using hostnames for consistent access &mdash; To ensure uninterrupted access, we recommend that you dbt Cloud services using hostnames. Hostnames provide a consistent reference point, regardless of any changes in underlying IP addresses. We are aligning with an industry-standard practice employed by organizations such as Snowflake.
 
 * Optimizing VPN connections &mdash; You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, you can direct traffic through the VPN and then to dbt Cloud. It's crucial to set up the proxy if you need to integrate with additional services.
+
+## API Access URLs
+
+dbt Cloud accounts with cell-based account prefixes have unique access URLs for account APIs. These URLs can be found in your **Account settings** below the **Account information** pane.
+
+<Lightbox src="/img/docs/dbt-cloud/access-urls.png" title="Access URLs in the account settings" />
+
+These URLs are unique to each account and begin with the same prefix as the URL used to [access your account](#accessing-your-account). The URLs cover the following APIs:
```

Comment:
In the screenshot, the Semantic Layer JDBC URL starts with `jdbc:arrow` not the same prefix as the others, Is this a typo or an outlier?
<img width="316" alt="Screenshot 2024-05-17 at 3 17 11 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/3880403/6161fa7a-6b54-4651-bdde-fb75df5c9a84">

Reply:
This is a unique format for the SL API